### PR TITLE
495 No feedback for domain validation

### LINKED ISSUE #495: No feedback for domain validation
Grabbed virtual appliance, setup port forwarding and run:
https://church.io/downloads/onebody/virtual-appliance/onebody-3.4.0-1.zip

At http://localhost:8080/setup, I entered no domain name. Instead of giving me feedback that I should enter a domain name when I clicked "save changes", all the form fields were simply erased.

### PR DESCRIPTION
**Pull request by toreyheinz**
Adds Setup spec to verify Person attributes get set and feedback is given for blank domain. Fixes #495

### CODE REVIEW

**Comment by seven1m** [comment]
@toreyheinz sorry I've been tuned out for a few days... is this ready for review? I saw commented-out code and thought maybe you were still working on it....

**Comment by toreyheinz** [comment]
@seven1m No worries. I just removed the pending spec, I think this is ready for review. I have some other updates I'd like to make, but just wanting to take some baby steps.

**Comment by seven1m** [comment]
Awesome, thanks!

File: spec/models/setup_spec.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+require_relative '../rails_helper'
+
+describe Setup do
+  let(:params) { ActionController::Parameters.new({
```
Comment (Tr4pSt3R) [inline spec/models/setup_spec.rb]: Think about moving this into a factory using FactoryGirl per this example here: (https://github.com/churchio/onebody/blob/master/spec/factories/prayer_requests.rb)

Comment (toreyheinz) [inline spec/models/setup_spec.rb]: Ah, now I see where the line note went.

**Comment by seven1m** [comment]
@toreyheinz sorry this took me so long to merge... Thanks for your work on this!!!

**Comment by andreiatcm** [comment]
No worries,

I just got back from a short term mission trip to Mexico, so I haven't been doing anything for about three weeks.

Hopefully I can keep picking away at things.

Torey

**Comment by toreyheinz** [comment]
Oops, I was logged in as @andreiatcm from Mexico, who I am helping to learn Rails.

**Comment by toreyheinz** [comment]
No worries,

I just got back from a short term mission trip to Mexico, so I haven't been doing anything for about three weeks.

Hopefully I can keep picking away at things.

Torey

**Comment by seven1m** [comment]
@toreyheinz glad to have you on the team! Let me know if you need anything!

**Pull request merged**